tests/discovery/test_discovery.py

```
@@ -40,19 +78,41 @@
     return link
 
 
-# wrap direct cases into signposted cases
+# Wrap direct cases into signposted cases
 SIGNPOSTED_CASES = list()
```

Ruff

Unnecessary `list` call (rewrite as a literal)

Rewrite as a literal

(C408)